packages/twenty-front/src/effect-components/WindowEventEffect.tsx

```diff
@@ -0,0 +1,50 @@
+import { useCallback, useEffect } from 'react';
+import { useSetRecoilState } from 'recoil';
+
+import { tokenPairState } from '@/auth/states/tokenPairState';
+import { REACT_APP_CHROME_EXTENSION_ID } from '~/config';
+import { useNavigate } from 'react-router-dom';
+
+export const WindowEventEffect = () => {
+  const navigate = useNavigate();
+  const setTokenPair = useSetRecoilState(tokenPairState);
+
+  const inIframe = useCallback(() => {
```

Comment:
Nice, useCallback is redundant here

Reply:
`isInIframe` Always begin function names with a verb

Reply:
Put it outside of the component if it's causing useEffect dependency issues because it's outside of the React context.

Reply:
You're right!